sdk/databox/Azure.ResourceManager.DataBox/src/Custom/Models/MitigateJobContent.cs

```diff
@@ -20,10 +22,20 @@ public MitigateJobContent()
         /// <param name="customerResolutionCode"> Resolution code for the job. </param>
         public MitigateJobContent(CustomerResolutionCode customerResolutionCode) : this()
         {
-            _customerResolutionCode = customerResolutionCode;
+            CustomerResolutionCode = customerResolutionCode;
         }
 
         /// <summary> Resolution code for the job. </summary>
-        public CustomerResolutionCode CustomerResolutionCode => _customerResolutionCode ?? default;
+        public CustomerResolutionCode CustomerResolutionCode
```

Comment:
the public serialization code will generate new internal ctor for this type which assigns value for this property.
Here we add a private set to ensure that could happen.
Due to the language version we are using in this repo, we cannot use `init` here because is is unavailable.